### PR TITLE
Pseudoqueue

### PR DESCRIPTION
**Pull request by paronicholas**
- [x] Top-level README “Table of Contents” is updated
- [x] Feature tasks for this challenge are completed
- [x] Unit tests written and passing
    - [x] “Happy Path” - Expected outcome
    - [x] Expected failure
    - [x] Edge Case (if applicable/obvious)
- [x] README for this challenge is complete
    - [x] Summary, Description, Approach & Efficiency, Solution, Resources
    - [x] Link to code
    - [x] Link to tests

### CODE REVIEW

File: code401challenges/src/main/java/stacksAndQueues/PseudoQueue.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      throw new NoSuchElementException();
+    } else {
+      looper(this.stack1, this.stack2);
+      T front = this.stack2.pop();
+      looper(this.stack2, this.stack1);
+      return front;
+    }
+  }
+
+  private void looper(Stack<T> stackToPopFrom, Stack<T> stackToPushInto) {
```
Comment (evanslaton) [inline code401challenges/src/main/java/stacksAndQueues/PseudoQueue.java]: Single Responsibility Principle > 9,000
